18F/identity-idp
# LG-10330 Make i18N into own Package

## 🎫 Ticket

[LG-10330 ](https://cm-jira.usa.gov/browse/LG-10330)  Make PO Search's no results view configurable

## 🛠 Summary of changes
- Bundle up i18n into it's own package (so that identity-sites can use it)


## 📜 Testing Plan
- Start idp locally. Go to a view that has translations and ensure that they are still working as expected. View all three langauges
- Import published package (i18n) into identity-sites and to see if translations are working as expected

![Screenshot 2023-09-12 at 12 38 36 PM](https://github.com/18F/identity-idp/assets/125507397/644131cc-0106-4940-a296-3115f04269ad)

![Screenshot 2023-09-12 at 12 38 47 PM](https://github.com/18F/identity-idp/assets/125507397/95beba20-9ddd-4537-ac79-6bfefec58d0c)

![Screenshot 2023-09-12 at 12 38 57 PM](https://github.com/18F/identity-idp/assets/125507397/8b28b150-3b2c-4eb2-aa08-b998c30e4f55)
